src/SuggestionsPopup.cpp

```diff
@@ -103,3 +105,41 @@ SuggestionsPopup::addUsers(const QVector<SearchResult> &users)
 
         resize(geometry().width(), 40 * users.size());
 }
+
+void
+SuggestionsPopup::cycleThroughSuggestions()
+{
+        auto item = layout_->itemAt(tab_clicks_);
+        if (!item) {
```

Comment:
You could merge those `if (!item)` statements into one by checking if the `tab_clicks_` is greater than the number of items in the layout. Or maybe by resetting the variable if it goes beyond that number at the end of the function.